workflow.go

```diff
@@ -279,9 +267,11 @@ func PublishURLs(ttlHashSet *ttl_hash_set.TTLHashSet, queueManager *queue.Manage
 			continue
 		}
 
-		if crawlCount == AlreadyCrawled {
-			log.Debugln("URL extracted from page already crawled:", url)
-		} else if crawlCount == NotRecentlyCrawled {
+		if crawlCount == Enqueued {
```

Comment:
Minor point, because it doesn't affect the functionality: this should be `crawlCount >= Enqueued`, otherwise we won't get the debug line if it's being retried.

Reply:
I have added a condition to log `crawlCount > Enqueued` with a slightly different message to help if we need to debug and updated the variable name to `enqueuedStatus` which I think is clearer.